napari/benchmarks/README.md

```diff
@@ -1,3 +1,23 @@
-These are benchmarks to be run with
-[asv](https://asv.readthedocs.io/en/stable/). They are not distributed with
+# Napari benchmarking with airspeed velocity (asv)
+
+These are benchmarks to be run with airspeed velocity 
+([asv](https://asv.readthedocs.io/en/stable/)). They are not distributed with
 installs.
+
+## Example commands
+
+Run all the benchmarks:
+
+`asv run`
+
+Do a "quick" run, where each benchmark function is run only once:
+
+`asv run --environment conda -q`
+
+To run a single benchmark (Vectors3DSuite.time_refresh) with the environment you are currently in:
+
+`asv run --python=same --bench Vectors3DSuite.time_refresh`
+
+Compare benchmarks across branches, run using conda environments (instead of vitualenv), and limit to the `Labels2DSuite` benchmarks:
```

Comment:
```suggestion
To compare benchmarks across branches, run using conda environments (instead of virtualenv), and limit to the `Labels2DSuite` benchmarks:
```